tasks/mock-github

```diff
@@ -40,6 +40,8 @@ class Handler(MockHandler):
             })
         elif self.path == f'/{repo}/{sha}/.cockpit-ci/container':
             self.replyData('ghcr.io/cockpit-project/tasks')
+        elif self.path == f'/repos/{repo}/contents/.cockpit-ci/container?ref={sha}':
+            self.replyData('ghcr.io/cockpit-project/tasks')
```

Comment:
That seems wrong.  Isn't it going to be expecting a JSON document?

Reply:
It is, I made a mess last night :)